samcli/cli/main.py

```diff
@@ -49,11 +50,115 @@ def print_info(ctx, param, value):
     if not value or ctx.resilient_parsing:
         return
 
-    click.echo(json.dumps({"version": __version__}, indent=2))
+    info = {
+        "version": __version__,
+        "system": _gather_system_info(),
+        "additional_dependencies": _gather_additional_dependencies_info(),
```

Comment:
Might be better to move "atleast" additional deps to another file.

Reply:
updated